node_modules/oae-authentication/lib/strategies/shibboleth/init.js

```diff
@@ -29,6 +29,7 @@ var AuthenticationUtil = require('oae-authentication/lib/util');
 var ShibbolethAPI = require('./api');
 var ShibbolethStrategy = require('./strategy');
 
+var AuthzConstants = require('oae-authz/lib/constants').AuthzConstants;
```

Comment:
We follow the following ordering when requiring dependencies:

1. All external dependencies
2. All OAE dependencies from different modules
3. All OAE dependencies from the current module

Therefore, `AuthzConstants` should be added to the `var Cassandra = ...` block.

Note that we order the requires alphabetically within their block.